src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -142,7 +142,13 @@ const WorkspaceContainer = ({
 
   useEffect(() => {
     if (isGoogleWorkspaceSyncing) {
-      Ajax().Metrics.captureEvent(Events.permissionsSynchronizationDelayDisplayed, extractWorkspaceDetails(workspace))
+      Ajax().Metrics.captureEvent(Events.permissionsSynchronizationDelayDisplayed,
+        _.merge(extractWorkspaceDetails(workspace),
```

Comment:
Note that you can also skip _.merge by using the spread operator. For example: https://github.com/DataBiosphere/terra-ui/blob/0d9f0e051c9a3a04a4ffee259dd17bc5e7af9cd3/src/pages/workspaces/workspace/analysis/modals/AnalysisDuplicator.ts#L77

Reply:
Is there a preference? I see both in the codebase.

Reply:
I personally try to avoid underscore functions when they aren't necessary… but I wouldn't say there's much of a preference.